Engine/Module/ModuleCamera.cpp

```diff
@@ -85,10 +86,13 @@ void ModuleCamera::SelectMainCamera()
 			if (main_camera == nullptr)
 			{
 				main_camera = camera;
+				canvas_camera = camera;
+				//scene_camera->SetClearMode(ComponentCamera::ClearMode::ORTHO);
 			}
 			else if (main_camera->depth < camera->depth)
 			{
 				main_camera = camera;
+				canvas_camera = camera;
```

Comment:
This logic should be inside `ModuleUI`, in order to have multiple viewports at the future.